google-cloud-compat-checker/README.md

```diff
@@ -0,0 +1,25 @@
+Google Cloud Java Compatibility Checker
```

Comment:
Then how about having a google-cloud-util module, and then put google-cloud-compat-checker under that?